tests/menubar-editor/change-state-of-menuitemcheckbox-in-submenu-using-reading-mode.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Change state of menu item checkbox to unchecked in reading mode</title>
+<link rel="author" title="Valerie Young" href="mailto:valerie@bocoup.com">
+<link rel="help" href="https://w3c.github.io/aria-practices/examples/menubar/menubar-2/menubar-2.html">
+<link rel="help" href="https://w3c.github.io/aria/#menuitemcheckbox">
+<link rel="help" href="https://w3c.github.io/aria/#aria-checked">
+
+<script type="module">
+  import { verifyATBehavior, displayTestPageAndInstructions } from "../resources/aria-at-harness.mjs";
+
+  verifyATBehavior({
+    
+    applies_to: ["Desktop Screen Readers"],
+    mode: ["reading"],
+    task: "change state of menuitemcheckbox in submenu using reading mode",
+    specific_user_instruction: "uncheck the 'Italic' menu item in the 'Style/Color' submenu",
+    output_assertions: [
+      [1, "The unchecked state of the checkbox is conveyed"],
+      [1, "Screen reader switching to interaction mode is conveyed"]
+    ]
+  });
+
+  displayTestPageAndInstructions("reference\menubar-editor.html");

```

Comment:
This blackslash needs to be updated to a forward slash, because this file is opened in a browser.